docs/extra/logs-collect.md

```diff
@@ -0,0 +1,23 @@
+# Collecting logs for Support
+
+To help troubleshoot issues, obtain the logs from your Codacy instance and send them to Codacy's Support:
+
+1.  Collect logs from your Codacy instance using Fluentd:
+
+    ```bash
+    --set fluentdoperator.enabled=true
+    ```
+
+    After executing this command the Fluentd daemonset will send the logs to MinIO, which is also installed by the Codacy chart.
```

Comment:
Do we need to  have this here? 
Since you are separating I would just keep a link here pointing to the installation saying you need that to have logs.

Reply:
Indeed, it felt like extraneous detail.

Since `fluentdoperator.enabled` is already true by default, I removed this entire instruction.